### ISSUE TITLE
Switch change handle?

### ISSUE THREAD

**Issue by safakseyrek** [InvexGames/MaterialUI#59]
How can i handle switch change ? Give me a sample code block ?

**Comment by atomhax** [comment]
<img width="281" alt="screen shot 2017-01-09 at 10 23 25 am" src="https://cloud.githubusercontent.com/assets/2883461/21776023/b6513a36-d655-11e6-9245-bf76e7fd2995.png">


Start here and see if you can't figure it out.

**Comment by safakseyrek** [comment]
Sorry , I have not seen this . Thanks 👍

**Comment by atomhax** [comment]
Good luck friend! Let me know if you need help.

**Comment by nah0y** [comment]
Hey guys, in case you didn't know. This github repo is super old, and an updated version of MaterialUI is available on the Asset Store and we're still working on it bringing updates etc... :)
https://www.assetstore.unity3d.com/#!/content/51870

**Comment by atomhax** [comment]
@nah0y Is it from the same author as this repo? I'd be willing to buy if it's the same author.

**Comment by InvexGames** [comment]
Yep, same author - @nahoy also works on the newer version with me :)
